Crickit_Stumblebot/code.py

```diff
@@ -0,0 +1,84 @@
+# Stumble Bot, coded in CircuitPython 
+# Using an Adafruit Circuit Playground Express, Crickit, and 2 servos
+# Dano Wall, Mike Barela for Adafruit Industries, MIT License, May, 2018
+#
+from digitalio import DigitalInOut, Direction, Pull
+from adafruit_seesaw.seesaw import Seesaw
+from adafruit_seesaw.pwmout import PWMOut
+
+from adafruit_motor import servo
+from busio import I2C
+import board
+import time
```

Comment:
Please move `import time` to before `from digitalio import DigitalInOut, Direction, Pull`.